data/example-data-oracle-book/src/main/resources/data-oracle-book-application.properties

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 American Express Travel Related Services Company, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+
+spring.oracle.datasource.username=
```

Comment:
should we split into two files

Reply:
Should I? I left it as is for now but I can move the reactive stuff to it's own property file. I kept it as one for now.